addons/nightvision/stringtable.xml

```diff
@@ -246,25 +247,30 @@
             <English>NVG Fog Scale</English>
             <Japanese>暗視装置の霧の規模</Japanese>
             <Italian>Livello Nebbia NVG</Italian>
+            <German>Nachtsicht Nebel Skala</German>
```

Comment:
`Nebel in Nachtsicht`

Das "scale" wird sich nicht auf "Skala", sondern eher auf das "Ausmaß" beziehen.